### ISSUE TITLE
allow transparent, reset or clear option

### ISSUE THREAD

**Issue by jklue** [sylvainjule/kirby-color-palette#28]
Sometimes when I choose a color I later realize I shouldn't have. I can reset as developer, but client should be able to undo color selection if possible. Thanks for any guidance here, and the amazing plugin!

**Comment by janstieler** [comment]
Yes or set a unchoosed swatch like a retancle with a red diagonal line like in some colorpickers.
